### ISSUE TITLE
Remove icon from the manifest

### ISSUE THREAD

**Issue by samskiter** [skimarxall/RealTextView#4]
I'm getting a manifest merge error related to the icon specified in the manifest.

**Comment by skimarxall** [comment]
Can you put the error here please

**Comment by samskiter** [comment]
I'll try an get a copy of the error for you if I can but you should be able to generate it by integrating the library into any project. You can basically remove the application tag from the manifest I think and remove the app icon images too.

**Comment by skimarxall** [comment]
Yes, probably I can do that, I'll see If I find time for it. Thanks

El dv., 6 març, 2015 00:51 Sam Duke <notifications@github.com> va escriure:

**Comment by crysan** [comment]
+100500

**Comment by rnascimento13** [comment]
i getting ic_launcher error too

**Comment by AndroidDeveloperLB** [comment]
I'm getthing this issue too:

Error:(52, 9) Attribute application@icon value=(@drawable/app_icon) from AndroidManifest.xml:52:9
	is also present at com.hardsoftstudio.real.textview:library:0.9.0:11:9 value=(@drawable/ic_launcher)
	Suggestion: add 'tools:replace="android:icon"' to <application> element at AndroidManifest.xml:49:5 to override

> Manifest merger failed : Attribute application@icon value=(@drawable/app_icon) from AndroidManifest.xml:52:9
	is also present at com.hardsoftstudio.real.textview:library:0.9.0:11:9 value=(@drawable/ic_launcher)
	Suggestion: add 'tools:replace="android:icon"' to <application> element at AndroidManifest.xml:49:5 to override

Any workaround for this?
